merlin-server/src/main/java/gov/nasa/jpl/aerie/merlin/server/remotes/postgres/PostgresResultsCellRepository.java

```diff
@@ -422,6 +422,7 @@ private static void postSimulatedActivities(
   public static final class PostgresResultsCell implements ResultsProtocol.OwnerRole {
     private final DataSource dataSource;
     private final SimulationRecord simulation;
+    private final PlanId planId;
```

Comment:
The PostgresResultsCellRepository serves as the boundary between the postgres models and the server models. Internally, we keep a `PlanId` instead of a `long` because we can easily get a long out of it when calling actions while simply passing the value in when a `PlanId` is required. However, if we used a `long` then every time we need a `PlanId` we would need to construct a new one from the `long`.

Reply:
Thank you for the clarification. That makes sense.